xir/validator.py

```diff
@@ -138,6 +146,14 @@ def run(self, raise_exception: bool = True) -> Optional[Sequence[str]]:
 
         return self._validation_messages or None
 
+    def _check_constants(self) -> None:
+        """Checks that constants are correctly declared."""
```

Comment:
In my opinion, the current docstring sounds like this function is checking the syntax of the `constants` block!
```suggestion
        """Checks that the declared constants are valid."""
```